src/rpk_entity.rs

```diff
@@ -0,0 +1,83 @@
+use crate::error::Error;
+use crate::signed_data::SubjectPublicKeyInfo;
+use crate::{der, signed_data, DerTypeId};
+use pki_types::{SignatureVerificationAlgorithm, SubjectPublicKeyInfoDer};
+
+/// A Raw Public Key, used for connections using raw public keys as specified in RFC7250.
```

Comment:
small nit: make life easier for the reader :-)

```suggestion
/// A Raw Public Key, used for connections using raw public keys as specified in [RFC 7250](https://www.rfc-editor.org/rfc/rfc7250).
```